### DISCUSSION TITLE
github pagesの設定画面が出ない。

### DISCUSSION THREAD

**Discussion by freegit897** [community/community#121073] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

GitHub Pagesで静的webサイトを公開する方法が分かりません。
というより所定の画面が出てこないです。下記の動画の説明によると1．リポジトリの作成、2．HTMLなどのファイルのアップロード（手動）、3．webページの公開となっている。2番までは既に済んでいます。問題は3番目です。画面右上の歯車のマーク｢setting｣をクリックしてしばらくスクロールすると｢gitgub pages｣があるのですが私の画面にはありません。下記の画像の右側を参照。動画ではDanger Zoneの上に｢gitgub pages｣がありますが私の場合、Danger Zoneの上に｢Archives｣がありお手本動画と全然違います。2番目までの過程は私がやったのと同じ方法です。2番まで同じ方法でやれたのに何で私の画面には｢gitgub pages｣がないのでしょうか？ ここから前に進めなくなりました。

https://www.youtube.com/watch?v=kftqAritHyc
![setting](https://github.com/community/community/assets/167831494/927315c2-79e2-4a7c-bf22-ec0539ca17cf)

**Comment by ebndev** [comment]
Hi @freegit897, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
